CalcIgor.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace CourseApp
+{
+    public class CalcIgor

```

Comment:
Rename class into Human Readability

Reply:
agree - class name does not reflects what he actually do